src/compiler/diagnosticMessages.json

```diff
@@ -2000,7 +2000,10 @@
         "category": "Error",
         "code": 2569
     },
-
+    "Property '{0}' does not exist on type '{1}'. Did you forget to await the '{1}'?": {

```

Comment:
how about `Property '{0}' does not exist on type '{1}'. Did you forget 'await' ?`

Reply:
@DanielRosenwasser can i get your input on this error message as well.

Reply:
`Property '{0}' does not exist on type '{1}'. Did you forget to use 'await'?` is probably the best one actually.

Reply:
I had this at https://github.com/Microsoft/TypeScript/pull/23058. So which message?

* Property '{0}' does not exist on type '{1}'. Did you forget to use 'await'?
* Property '{0}' does not exist on type '{1}', but does exist on type '{2}'. Did you forget to use 'await'?